### ISSUE TITLE
Handle in color area should stay at dragged position

### ISSUE THREAD

**Issue by bjarnef** [umbraco/Umbraco.UI#463]
### Which *exact* UI Library version are you using? For example: 1.0.1 - don't just write v1.

1.2.0

### Bug summary

I noticed the handle in color area doesn't work as expected here https://uui.umbraco.com/?path=/docs/uui-color-area--overview

There has been made changes since I was working on it, so not sure if any of these affect this.

When dragging handle to bottom (black) the handle jump to bottom left corner, but it should stay at the dragged position.

### Steps to reproduce

Drag handle to bottom of color area and notice the handle jump to bottom left corner.

**Comment by bjarnef** [comment]
@iOvergaard it seems this may be a calculation issue somewhere.
E.g. compare with color area here: https://shoelace.style/components/color-picker?id=inline

**Comment by julczka** [comment]
Hello everyone! The jumping handle happens when you drag it to the bottom of the area. The very bottom part (that would be 0 on Y axix) means that the color is equal to black. On the X axis the colors saturation is reflected. Now: the value for of saturation for black is 0,  and [the handle position is binded to the saturation ](https://github.com/umbraco/Umbraco.UI/blob/c00c5f5af5c6d7e240d1a44f9109cf9e3ba45169/packages/uui-color-area/lib/uui-color-area.element.ts#L269) and therefore the handle jumps to the left bottom corner when the value is black. I think the solution here could be caching the previous x position if the handle with each move and if lightness (Y axis) quals to 0 the X axix position should be equal to the previous one od incremented/decremented accordigly to the movement.

**Comment by bjarnef** [comment]
@julczka it doesn't happen in the color picker here: https://shoelace.style/components/color-picker
https://github.com/shoelace-style/shoelace/blob/be1c38f0e52bc784241f4b6358a8c76cc912d55a/src/components/color-picker/color-picker.ts#L797

Furthermore the color area doesn't work as expected when format is initially set to "hsl":
https://uui.umbraco.com/?path=/story/uui-color-picker--formats

**Comment by bjarnef** [comment]
@julczka @iOvergaard besides the object value of alpha property, it seems it was an issue with set of `value` property in `syncValues()` function.

with `Colord` it didn't seem to like parsing the HSV color string as with TinyColor:
https://github.com/shoelace-style/shoelace/blob/current/src/components/color-picker/color-picker.ts#L654

```
`hsva(${hue}, ${saturation}, ${brightness}, ${alpha / 100})` 
```

so I used HSL instead:
```
`hsla(${hue}, ${saturation}%, ${lightness}%, ${alpha / 100})`
```

**Comment by iOvergaard** [comment]
Should be fixed with #486 and will be released in 1.3.0. We'll aim to have another RC out next week.

**Comment by bjarnef** [comment]
@iOvergaard I wonder why color area has the `alpha` property?

and only adjusting `value` property has impact on rendering in storybook:

![image](https://github.com/umbraco/Umbraco.UI/assets/2919859/65cef524-cd7e-4396-be31-0f7b3a9ef0e9)

Furthermore I get this when browsing e.g. color picker component:

![image](https://github.com/umbraco/Umbraco.UI/assets/2919859/79d9978d-df08-407d-996d-b22dad00fa3e)
